project-p2-layouts-master/code/index.html

```diff
@@ -0,0 +1,198 @@
+        </div>
+        <div class="question-beige">
+          <img src="#" alt="" >
+        </div>
+        <div class="question-blue">
+          <img src="#" alt="" >
+        </div>
+        <div class="question-beige">
+          <img src="#" alt="" >
+        </div>
+        <div class="question-blue">
+          <img src="#" alt="" >
+        </div>
+        <div class="question-beige">
+          <img src="#" alt="" >
+        </div>
+        <div class="question-blue">
+          <img src="#" alt="" >
+        </div>
+        <div class="question-beige">
+          <img src="#" alt="" >
+        </div>
+        <div class="question-blue">
+          <img src="#" alt="" >
+        </div>
+        <div class="question-beige">
+          <img src="#" alt="" >
+        </div>
+      </section>
+
+      <div class="subscribe"> 
+        <h3>Interessiert?</h3>
+        <p>Möchtest du regelmässig über Neuigkeiten von Jobsurfing.ch informiert werden?</p>
+        <a href="#">
+          <button>Subscribe</button>
+        </a>
+      </div>
+
+      <div class="subtitle">
+        <h2>In 5 Schritten zu mehr Klarheit</h2>
+     </div>
+
+      <section class="steps">
+        <div class="step">
+          <div class="stepnumber">1.</div>
+          <div class="stepicon">
+            <img src="Bilder/step1_Gerd Altmann-Pixabay.jpg" alt="Laptop-mobile">
+          </div>
+          <div class="steptext">
+            <p>Account erstellen</p>
+          </div>
+        </div>
+        <div class="step">
+          <div class="stepnumber">2.</div>
+          <div class="stepicon">
+            <img src="Bilder/Step2_GerdAltmann_Pixabay.jpg" alt="finger-points-on-boxes">
+          </div>
+          <div class="steptext">
+            <p>Deine Zielbranchen auswählen</p>
+          </div>
+        </div>
+        <div class="step">
+          <div class="stepnumber">3.</div>
+          <div class="stepicon">
+            <img src="Bilder/step3_Andrea_Piacquadio_Pexels.jpg" alt="">
+          </div>
+          <div class="steptext">
+            <p>Unternehmen kontaktieren
+            </p>
+          </div>
+        </div>
+        <div class="step">
+          <div class="stepnumber">4.</div>
+          <div class="stepicon">
+            <img src="Bilder/step4_pexels-ketut-subiyanto-4350036.jpg" alt="">
+          </div>
+          <div class="steptext">
+            <p>Jobsurfing: lernen und Klarheit gewinnen</p>
+          </div>
+        </div>
+        <div class="step">
+          <div class="stepnumber">5.</div>
+          <div class="stepicon">
+            <img src="Bilder/step5_mohamed_ Hassan_Pixabay.jpg" alt="Tablet-five-stars">
+          </div>
+          <div class="steptext">
+            <p>Referenz schreiben und erhalten</p>
+          </div>
+        </div>
+      </section>
+
+      <div class="subtitle">
+        <h2>Testimonials </h2>
+     </div>
+
+      <section class="testimonials">
+
+      </section>
```

Comment:
Remove unused code before submitting. Although - I'm guessing you will fill this site with content later on?